apps/web/src/pages/Applications/ApplicationWelcomePage/ApplicationWelcomePage.tsx

```diff
@@ -113,13 +119,23 @@ const ApplicationWelcome = ({ application }: ApplicationPageProps) => {
         )}
       </p>
       <p data-h2-margin="base(x1, 0)">
-        {intl.formatMessage({
-          defaultMessage:
-            "The GC Digital Talent platform is a skills-based hiring system. This means that your application will put a heavier focus on your skills and how you've used them in past experiences to help us get a stronger understanding of your fit.",
-          id: "u/DBSl",
-          description:
-            "Description of how the skills-based hiring platform assess candidates.",
-        })}
+        {intl.formatMessage(welcomeMessages.description)}
+        {/* Method 2 below: */}
+        {/* {isPublishedByIAP
+          ? intl.formatMessage({
+              defaultMessage:
+                "The Program is a Government of Canada initiative specifically for First Nations, Inuit, and Métis peoples. It is pathway to employment in the federal public service for Indigenous peoples who have a passion for Information Technology (IT). We focus on that passion, and your potential to grow and succeed in this field.",
+              id: "eyI7JA",
+              description:
+                "Description of how the skills-based hiring platform assess candidates.",
+            })
+          : intl.formatMessage({
+              defaultMessage:
+                "The GC Digital Talent platform is a skills-based hiring system. This means that your application will put a heavier focus on your skills and how you've used them in past experiences to help us get a stronger understanding of your fit.",
+              id: "u/DBSl",
+              description:
+                "Description of how the skills-based hiring platform assess candidates.",
+            })} */}
```

Comment:
I think this method would be preferred for now until we need different text, then we could easily convert to the second method with some tweaks.

Also, we introduced a context provider for this exact thing. You can do the following and it should work:

```tsx
const { isIAP } = useApplicationContext();
```

Reply:
Okay, I'll you this method for now until we decide to add more copy. Thanks!